h2o-genmodel/src/main/java/hex/genmodel/easy/DomainMapConstructor.java

```diff
@@ -0,0 +1,35 @@
+package hex.genmodel.easy;
+
+import hex.genmodel.GenModel;
+
+import java.util.HashMap;
+
+/**
+ *  Create map from input variable domain information.
+ */
+public class DomainMapConstructor {
+
```

Comment:
static methods are not ALWAYS bad: `function(input) -> output` with no side effect = perfect candidate for a static method: I don't really see the benefit of the "kingdom of nouns" here.

Reply:
@seb-h2o Agree, was also thinking that probably it is too much.... but decided that it is not bounded to the scope of what is called EasyPredictModelWrapper. I got used to prefer classes over static methods because it is possible to mock/extend them during testing. Also I think that we have enough static methods on the project :)

Reply:
@seb-h2o true, they are not bad until you want to unit test the code that uses them.

Reply:
@honzasterba that's why I mentioned "with no side-effect": there's no reason to mock pure functions when you want to test caller.

Reply:
@seb-h2o side effects are not the only reasons for using mocks. For the sake of testing I might want a specific output to be returned from this method. But it is definitely  true that we are abusing ability to mutate everything around our methods in Java.